ReleaseProcessAutomation.UnitTests/SemanticVersioning/SemanticVersionTests.cs

```diff
@@ -42,7 +42,86 @@ public void GetNextPossibleVersionsDevelop_WithPreReleaseAlpha_ReturnsSixVersion
   }
 
   [Test]
-  public void GetNextPossibleVersionsHotfix_WithNonPreRelease_ReturnsVersions ()
+  public void GetNextPossibleVersionsDevelop_WithPreReleaseRC_ReturnsExpectedVersions()
+  {
+    var parser = new SemanticVersionParser();
+    var expectedVersions = new SemanticVersion[]
+                           {
+                               parser.ParseVersion("1.3.0-rc.2"),
+                               parser.ParseVersion("1.3.0"),
+                               parser.ParseVersion("2.0.0-alpha.1"),
+                               parser.ParseVersion("2.0.0-beta.1"),
+                               parser.ParseVersion("2.0.0")
+                           };
+    var baseVersion = parser.ParseVersion("1.3.0-rc.1");
+
+    var possibleVersions = baseVersion.GetNextPossibleVersionsDevelop();
+
+    Assert.That(possibleVersions, Is.EqualTo(expectedVersions));
+  }
+  
+  [Test]
+  public void GetNextPossibleVersionsDevelop_WithMajorVersionWithoutPreRelease_ReturnsExpectedVersions()
```

Comment:
I can't spot the difference between this test and GetNextPossibleVersionsDevelop_WithMajorVersion_ReturnsExpectedVersions

(i.e. why is the list of expected versions different, the rest seems identical to me. please spot the difference for me ;) )

Reply:
`GetNextPossibleVersionsDevelop(bool withoutPreRelease = false)`
We are testing the difference between having the method suggest all versions or only the ones that aren't prereleases.